tests/container_hardening/test_container_hardening.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+import logging
+from tests.common.helpers.assertions import pytest_assert
+
+pytestmark = [
+    pytest.mark.topology('any'),
+]
+
+logger = logging.getLogger(__name__)
+
+
+def test_bgp_dev(duthost):
+    """
+    Test bgp container has no access to /dev/vda* or /dev/sda*
+    """
+    device = duthost.shell("docker exec bgp bash -c 'df -h | grep /etc/hosts' | awk '{print $1}'")['stdout']
```

Comment:
>device [](http://example.com/codeflow?start=4&length=6)

Will you check `shell` return code, and device is a valid path? If not, there may be risk to be inject into another command.

Reply:
I checked return code and valid path for device